### PR TITLE
Icon for `k-stat`

### PR DESCRIPTION
**Pull request by distantnative**
### Enhancement
- `k-stat` supports new `icon` prop

Used in System view:
Screenshot 2023-11-02 at 22 46 46

### CODE REVIEW

**Comment by bastianallgeier** [comment]
TBH, the icons don't work for me on a non-retina screen in this size. They are barely recognizable

File: panel/src/components/Layout/Stat.vue [hunk 1]
```diff
@@ -111,7 +108,13 @@ export default {
 	margin-bottom: var(--spacing-1);
 }
 .k-stat-label {
+	--icon-size: var(--text-xs);
```
Comment (afbora) [inline panel/src/components/Layout/Stat.vue]: So tiny for me. IMO `--text-md` looks great 👍

**Comment by distantnative** [comment]
That's only because you have so giant screens 🤡

I think `--text-md` doesn't go well with the label font size anymore, but `--text-sm` works for me
Screenshot 2023-11-03 at 16 04 02
Screenshot 2023-11-03 at 16 04 40

**Review by afbora — APPROVED** [review]
LGTM

**Pull request merged**